GitLab/README.md

```diff
@@ -0,0 +1,38 @@
+# Nimbella Commander GitLab Command Set
+A Nimbella Commander command set for viewing GitLab project statistics, creating tickets, and assigning issues. Currently supports slack.
+
+## Features
+- View community contribution stats for GitLab projects
+- Create a ticket for a GitLab project
+- Assign issues for a GitLab project
+- View list of users in project repository
+
+## Install
+```
+/nc csm_install xxx
+```
+
+## Requirements
+```
+In order to use this command set, you must set up a Personal Access Token on GitLab with API access. And save the token as a secret with the key 'AcessToken_GitLab'
+https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html
```

Comment:
Please also mention that the key needs to be added by running `/nc secret_create`

Reply:
I'll keep this in mind, thank you